client/package.json

```diff
@@ -8,7 +8,9 @@
     "@testing-library/user-event": "^12.6.0",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "react-scripts": "4.0.1",
+    "react-native": "^0.63.4",
```

Comment:
Why have you added these libraries ? dont think these are required

Reply:
react-scripts module was needed to run the client side

Reply:
I am saying about the newly added libraries such as react-scroll and react-native

Reply:
yes sir those are not-needed
i was trying something else